pkg/apic/service.go

```diff
@@ -25,6 +25,13 @@ const (
 	deleteAPI            = iota
 )
 
+const (
+	apiServerPageSize    = 20
+	healthcheckEndpoint  = "central"
+	attributesQueryParam = "attributes."
+	apiServerFields      = "name,title,attributes"
+)
+
```

Comment:
not sure why these showed up here. I only see the pagesize referenced in any of your changes. I also see these exact defs in discoverycache.go. Seems like we shouldn't have an exact copy of them in 2 places.

Reply:
yeah, will update, i forgot to remove.